airbyte-integrations/connectors/source-mongodb-internal-poc/src/main/java/io/airbyte/integrations/source/mongodb/internal/cdc/MongoDbCdcStateHandler.java

```diff
@@ -40,12 +40,15 @@ public AirbyteMessage saveState(final Map<String, String> offset, String dbHisto
 
   @Override
   public AirbyteMessage saveStateAfterCompletionOfSnapshotOfNewStreams() {
-    throw new RuntimeException("Debezium is not used to carry out the snapshot of tables.");
+    LOGGER.info("Snapshot of new collections is complete, saving state...");
```

Comment:
This should keep throwing `throw new RuntimeException("Debezium is not used to carry out the snapshot of tables.");` cause we are not using Debezium to carry out snpshot of tables

Reply:
Will fix.